docker/nginx/nginx-key.conf

```diff
@@ -1,18 +1,18 @@
-#
-# Copyright (c) 2018
-# Mainflux
-#
+# Copyright (c) Mainflux
 # SPDX-License-Identifier: Apache-2.0
-#
+
 # This is the default Mainflux NGINX configuration.
 
 user nginx;
 worker_processes auto;
+worker_cpu_affinity auto;
 pid /run/nginx.pid;
 include /etc/nginx/modules-enabled/*.conf;
 
 events {
-    worker_connections 768;
+    # Explanation: https://serverfault.com/questions/787919/optimal-value-for-nginx-worker-connections
+    # We'll keep 1M connections per core (assuming one worker per core)
+    worker_connections 1000000;
```

Comment:
I know, but with 10k we already hit the limitation even for simple test of 1000 publishers publishing each 10 messages.

This is not a production set-up, but I wanted to avoid at least that this limit us do the tests.

Reply:
Fixed to follow reccomendation, just in case. If needed can be bumped up for tests, but I guess limitations will first come from Linux TCP buffers and otehr stuff...